sync3/lists.go

```diff
@@ -153,8 +166,8 @@ func (s *InternalRequestLists) Get(listKey string) *FilteredSortableRooms {
 func (s *InternalRequestLists) ListsByVisibleRoomIDs(muxedReqLists map[string]RequestList) map[string][]string {
 	listsByRoomIDs := make(map[string][]string, len(muxedReqLists))
 	// Loop over each list, and mark each room in its sliding window as being visible in this list.
-	for listName, reqList := range muxedReqLists {
-		sortedRooms := s.lists[listName].SortableRooms
+	for listKey, reqList := range muxedReqLists {
```

Comment:
This isn't strictly relevant, but all the existing code uses `listKey`, apart from this bit that I wrote.